app/services/etherscan_service.rb

```diff
@@ -13,10 +13,30 @@ def is_proof_member?(address)
       address: address
     )
 
-    if response.dig('result')&.last&.dig('to')&.downcase == address.downcase
+    transactions = response.dig('result')
+
+    inCount = 0
+    outCount = 0
+
+    transactions.each do |child|
+      toAddress = child.dig('to').downcase
+      fromAddress = child.dig('from').downcase
+
+
+      if toAddress == address.downcase
+        inCount += 1
+      end
+      if fromAddress == address.downcase
+        outCount += 1
+      end
+    end
+
+    if inCount > outCount
       true
+      print 'true'
     else
       false
+      print 'false'
```

Comment:
you can remove these `print` lines on 36 and 39... ruby always returns the return value of the last line executed in a method so it will need to be `true` or `false` in this case. If you remove 36 and 39 this looks good.

You could also consider making this a one line ternary `in_count > out_count ? true : false`